sysutils/cdrtools/Portfile

```diff
@@ -1,98 +1,211 @@
 
-PortSystem          1.0
-PortGroup           compiler_blacklist_versions 1.0
-
-name                cdrtools
-version             3.01
-revision            1
-categories          sysutils
-# listing all these licenses is somewhat redundant when taken as a whole,
-# but different separable components are licensed differently
-license             CDDL-1 BSD LGPL-2.1 GPL-2+ GPL-2
-platforms           darwin
-conflicts           dvdrtools
-
-description         ISO 9660 filesystem and CD creation tools
-long_description    The cdrtools software includes programs to create \
-                    and/or extract ISO 9660 filesystems, verify their \
-                    integrity, and write them to a disc.
-homepage            http://cdrecord.org
-maintainers         nomaintainer
-master_sites        sourceforge:project/cdrtools
-
-checksums           rmd160  3696b1bc502905cbe7c06e492b42d2ffe2dba982 \
-                    sha256  ed282eb6276c4154ce6a0b5dee0bdb81940d0cbbfc7d03f769c4735ef5f5860f \
-                    size    2087416
-
-use_bzip2           yes
-
-depends_build       port:smake
-
-depends_lib         port:gettext
-
-patchfiles          patch-include_schily_sha2.h
-patchfiles-append   mkisofs.c.patch
-post-patch {
-    reinplace -locale C "s|/opt/schily|${prefix}|g" \
-        ${worksrcpath}/DEFAULTS/Defaults.darwin \
-        ${worksrcpath}/DEFAULTS/Defaults.mac-os10 \
-        ${worksrcpath}/DEFAULTS_ENG/Defaults.darwin \
-        ${worksrcpath}/DEFAULTS_ENG/Defaults.mac-os10 \
-        ${worksrcpath}/libfind/find.c \
-        ${worksrcpath}/libfind/find_main.c \
-        ${worksrcpath}/librscg/scsi-remote.c \
-        ${worksrcpath}/TEMPLATES/Defaults.gcc
+PortSystem              1.0
+PortGroup               makefile 1.0
+PortGroup               compiler_blacklist_versions 1.0
+
+name                    cdrtools
+version                 3.02
+revision                0
+livecheck.version       2023-04-19
+categories              sysutils
+platforms               darwin
+conflicts               dvdrtools
+maintainers             {hotmail.com:amtor @RobK88} \
+                        openmaintainer
+license                 CDDL-1 BSD LGPL-2.1 GPL-2+ GPL-2
+description             ISO 9660 filesystem and CD creation tools
+long_description        The cdrtools software includes programs to create \
+                        and/or extract ISO 9660 filesystems, verify their \
+                        integrity, and write them to a disc.
+
+homepage                https://codeberg.org/schilytools/schilytools
+
+#
+# Need a recent version of git to fetch the source while developing this port
+# on my old Mac running High Sierra.
+# Otherwise the fetch will fail with a SSL certificate expired error.
+#
+# Will switch to a standard tarball download for this port once development has finished.
+#
+
+depends_fetch           port:git
+
+fetch.type              git
```

Comment:
You don't need it here. I suggest to use technic similar with https://github.com/macports/macports-ports/pull/18655

Reply:
@catap - Since I now know how I can test this port locally on my Mac running High Sierra to overcome the Lets Encrypt cert expired bug (see https://trac.macports.org/wiki/ProblemHotlist), I have just updated this port, `cdrtools` to download the source tarball instead of using `git` as the fetch method.